nss-day-cohort-60/national-parks-client
# Jd add blog by park filter

# Description
finished implementing all blog search and filter functions

# Steps to test

1. Fetch the JD-add-blog-by-park-filter branch and switch to it
2. Start/restart debugger
3. navigate to blog page and test dropdown filters by park
4. Entries in the search field should further reduce blogs according to search term
5. Check that search works when all blogs are selected also
